CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing 🙌
+
+> When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+So you want to be a part of "The Compose Revolution" and that's great. Here are some contribution opportunities, in order of difficulty.
```

Comment:
It's somewhat emotional for an official document.

Reply:
Okay. What do you suggest?

Reply:
Are you looking for ways to contribute to Compose? Here are some opportunities you can explore, in order of difficulty.

Reply:
That sounds better. Will update.

Reply:
Updated via https://github.com/JetBrains/compose-jb/pull/511/commits/59f6636402682a1f8825ccfc0a74c2bc0b8c3813